src/frontend/shortlist/src/api/shortlist.js

```diff
@@ -36,10 +36,37 @@ export default class ShortlistApi {
   authLogin(payload, successCb, failCb) {
     return new tempAuthLogin(payload, successCb, failCb);
   }
+  removeSchoolCard(payload, successCb, failCb) {
+    return new tempRemoveSchoolCard(payload, successCb, failCb);
+  }
 }
 
 import axios from "axios";
 import md5 from "md5";
+
+export class tempRemoveSchoolCard {
+  constructor(payload, successCb, failCb) {
+    // this.recoID = payload.recoID;
+    // this.trashed = payload.trashed;
+    this.payload = payload;
+    this.successCb = successCb;
+    this.failCb = failCb;
+  }
+  execute() {
+    axios({
+      method: "POST",
+      url: "https://api.shortlist.nyc/recommendation/update",
+      headers: {},
+      data: {
+        recoID: this.recoID,
```

Comment:
You would have to either define `recoID` and `trashed` above in the constructor or use `payload` here. If you use payload you would be able to reuse this function for marking as accepted, and just pass in different parameters as part of the payload. Otherwise you can have different functions for markTrashed and markAccepted.

Reply:
I see. I think in the previous pr, since we were both changing things at the same time, so I thought this is what you meant.

Reply:
> You would have to either define `recoID` and `trashed` above in the constructor or use `payload` here. If you use payload you would be able to reuse this function for marking as accepted, and just pass in different parameters as part of the payload. Otherwise you can have different functions for markTrashed and markAccepted.
> 
> Edit: it may be easier if you merge the changes in this branch and the ones in mark-school-as-accepted as they have a decent amount of overlap.

I don't think they have much overlap? They just have similar structure, but different functions? I though separating them would be easier for me to debug, like first merge one and see if it works as expected and passing all the tests etc.